tests/test_qe_app.py

```diff
@@ -11,29 +13,35 @@ def test_notebook_service_available(notebook_service):
     assert response.status_code == 200
 
 
-def test_qe_app_take_screenshot(selenium_driver, screenshot_dir):
+def test_qe_app_take_screenshot(selenium_driver, final_screenshot):
     driver = selenium_driver("qe.ipynb", wait_time=30.0)
     driver.set_window_size(1920, 1485)
     time.sleep(15)
-    driver.get_screenshot_as_file(str(Path.joinpath(screenshot_dir, "qe-app.png")))
 
 
-def test_qe_app_select_silicon(selenium_driver, screenshot_dir):
+def test_qe_app_select_silicon_and_confirm(
+    selenium_driver,
+    screenshot_dir,
+    final_screenshot,
+):
     driver = selenium_driver("qe.ipynb", wait_time=30.0)
     driver.set_window_size(1920, 1485)
-    driver.find_element(
-        By.XPATH, "//*[text()='From Examples']"
-    ).click()  # click `From Examples` tab for input structure
+
+    element = WebDriverWait(driver, 60).until(
```

Comment:
```suggestion
    element = WebDriverWait(driver, 10).until(
```
I'd say 60 is an overkill (same below) but up to you.

Reply:
Fair, 60 is too much.